go.mod

```diff
@@ -18,7 +18,7 @@ require (
 	github.com/emicklei/proto v1.6.15
 	github.com/getkin/kin-openapi v0.1.1-0.20190507152207-d3180292eead
 	github.com/ghodss/yaml v1.0.0
-	github.com/gogo/protobuf v1.3.0
+	github.com/gogo/protobuf v1.3.1
```

Comment:
master is 1.3.0. Do we want to have 1.4 higher than master?

Reply:
You are right. We don't need 1.3.1. Fixed.